pkgs/development/python-modules/generic/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, python3
```

Comment:
```suggestion
```

This is strictly forbidden in pythonPackages. Please list all the inputs that are under python3 and python3.pkgs.

Reply:
Solved